packages/expo-location/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 ### 🐛 Bug fixes
 
+- [iOS] Added error handler to the streaming location/heading methods since these can fail while streaming ([]() by [@chrfalch](https://github.com/chrfalch)) ([#35004](https://github.com/expo/expo/pull/35004) by [@chrfalch](https://github.com/chrfalch))
```

Comment:
```suggestion
- [iOS] Added error handler to the streaming location/heading methods since these can fail while streaming ([#35004](https://github.com/expo/expo/pull/35004) by [@chrfalch](https://github.com/chrfalch))
```